### PR TITLE
Absorption spectrum with Gaussian lineshape

### PR DESCRIPTION
**Pull request by baumeier**
Implements #2

### CODE REVIEW

**Comment by felipeZ** [comment]
@baumeier  instead of having hard-coded constant like `27.211`, we can use the predefined constant in scipy like:
```py
from scipy.constants import physical_constants 
 h2ev = physical_constants['Hartree energy in eV'][0]
```

**Comment by baumeier** [comment]
@felipeZ can you have another look?

**Comment by felipeZ** [comment]
@baumeier also `scipy` should be added to the dependencies in the `setup.py` file

File: PyVOTCA/utils.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+"""Utilities and constants."""
+from scipy.constants import physical_constants
+
+
+class Utils:
```
[the remark this answers is not in the document]
Comment (baumeier) [reply PyVOTCA/utils.py]: Changed.

File: PyVOTCA/visualization.py [hunk 2]
```diff
@@ -16,3 +17,37 @@ def plotQPcorrections(self):
...
+        plt.plot(e,spectrum,'k',linewidth=2)
+        plt.ylim(bottom=0)
+        plt.title(f'Gaussian lineshape with sigma = {sigma}eV')
+        plt.xlabel('Energy (eV)')
+        plt.ylabel('Absorption (arb. units)')
+        plt.show()
+
+    def gaussian(self, x, mu, sig):
+        #ATTN: not normalized
+        return np.exp(-np.power((x - mu)/sig, 2.)/2)
```
[the remark this answers is not in the document]
Comment (baumeier) [reply PyVOTCA/visualization.py]: Changed.

**Comment by baumeier** [comment]
> @baumeier also `scipy` should be added to the dependencies in the `setup.py` file

Done.

**Review by felipeZ — APPROVED** [review]

**Comment by felipeZ** [comment]
@baumeier  I think that we are good to go!

**Pull request merged**